ansible/roles/ocp-workload-gogs/tasks/workload.yml

```diff
@@ -0,0 +1,89 @@
+---
+# Implement your Workload deployment tasks here
+
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+- name: Create Project {{ gogs_project }}
+  k8s:
+    state: present
+    definition: "{{ lookup('template', item ) | from_yaml }}"
+  loop:
+  - ./templates/project.j2
+  register: r_createproject
+  until: r_createproject is succeeded
+  retries: 5
+
+- name: Get apps route suffix
```

Comment:
I just submitted a PR #836 for the ocp-workload-terminal role to change that logic since it keeps failing when deleting the dummy route for some reason. You don't actually need that dummy route. You can just create the real route and then retrieve it.

If you only need to support OCP 4 there is an even easier way:
https://github.com/redhat-gpte-devopsautomation/ansible-operator-roles/blob/master/roles/gogs-ocp/tasks/main.yml#L16

Reply:
Changed